lighthouse-core/formatters/accessibility.js

```diff
@@ -50,6 +50,14 @@ class Accessibilty extends Formatter {
         throw new Error('Unknown formatter type');
     }
   }
+
+  static getHelpers() {
```

Comment:
> is this a magic handlebars thing?

magic [report-generator thing](https://github.com/GoogleChrome/lighthouse/blob/e43341eb6a5de0ee79e33bd73acc44f8aca6d728/lighthouse-core/report/report-generator.js#L241), getting any helpers from the formatter before use